package.json

```diff
@@ -0,0 +1,22 @@
+{
```

Comment:
This file should be in `tests/Application`. It is useless in plugin root.

Compare it with old one in `tests/Application` and decide what changes we adopt, or even replace old one by this new one.